### PR TITLE
Change veth pair ip

### PR DESCRIPTION
**Pull request by Nino-K**
Changes the IP Addresses for the veth pair to reduce collision with users' existing network.

Fixes: https://github.com/rancher-sandbox/rancher-desktop/issues/7383

### CODE REVIEW

File: src/go/networking/cmd/proxy/wsl_integration_linux.go [hunk 1]
```diff
@@ -37,7 +37,7 @@ var (
 const (
 	defaultLogPath = "/var/log/wsl-proxy.log"
 	defaultSocket  = "/run/wsl-proxy.sock"
-	bridgeIPAddr   = "192.168.1.2"
+	bridgeIPAddr   = "192.168.143.2"
```
Comment (mook-as) [inline src/go/networking/cmd/proxy/wsl_integration_linux.go:40]: So if I understand this correctly, it used to talk to the namespace veth, but it now talks to the WSL veth?

**Review by mook-as — COMMENTED** [review]
Not quite sure this is correct; going to run in and poke at the live environment then approve if everything works fine.

**Review by mook-as — CHANGES_REQUESTED** [review]
I just realized that at a minimum we still have to edit the PR for missing some instances of the IP address:

https://github.com/rancher-sandbox/rancher-desktop/blob/cced1b8e375e2dfcb43c3d1b22f19a463edc81fc/bats/tests/registry/creds.bats#L44
https://github.com/rancher-sandbox/rancher-desktop/blob/cced1b8e375e2dfcb43c3d1b22f19a463edc81fc/docs/networking/windows/rancher-desktop-networking.md#L114
https://github.com/rancher-sandbox/rancher-desktop/blob/cced1b8e375e2dfcb43c3d1b22f19a463edc81fc/pkg/rancher-desktop/backend/wsl.ts#L1364

And since we inverted `veth-rd0` and `veth-rd0`, we need to update the docs:
https://github.com/rancher-sandbox/rancher-desktop/blob/cced1b8e375e2dfcb43c3d1b22f19a463edc81fc/docs/networking/windows/rancher-desktop-networking.md?plain=1#L23
https://github.com/rancher-sandbox/rancher-desktop/blob/cced1b8e375e2dfcb43c3d1b22f19a463edc81fc/docs/networking/windows/rancher-desktop-networking.md?plain=1#L30

**Comment by Nino-K** [comment]
> Not quite sure this is correct; going to run in and poke at the live environment then approve if everything works fine.

I toggled the veth pair around, I will rename it for clarity.

File: src/go/networking/cmd/proxy/wsl_integration_linux.go (hunk 1 above)
Comment (mook-as) [reply src/go/networking/cmd/proxy/wsl_integration_linux.go:40]: Yeah, this is the wrong address; it needs to talk to the namespace veth instead.

```
time="2024-09-12T12:16:30-07:00" level=debug msg="received the following port: [80] from portMapping: {Remove:false Ports:map[80/tcp:[{HostIP:0.0.0.0 HostPort:80}]] ConnectAddrs:[]}"
time="2024-09-12T12:16:56-07:00" level=debug msg="port proxy accepted connection from 127.0.0.1:53706"
time="2024-09-12T12:16:56-07:00" level=error msg="Failed to dial upstream 192.168.143.2:80: dial tcp 192.168.143.2:80: connect: connection refused"
```
```
# curl --head 192.168.143.1
HTTP/1.1 200 OK
Server: nginx/1.26.2
Date: Thu, 12 Sep 2024 19:18:38 GMT
Content-Type: text/html
Content-Length: 615
Last-Modified: Mon, 12 Aug 2024 14:28:31 GMT
Connection: keep-alive
ETag: "66ba1c0f-267"
Accept-Ranges: bytes
```

Comment (Nino-K) [reply src/go/networking/cmd/proxy/wsl_integration_linux.go:40]: Yes, I missed the last octet.

File: src/go/networking/README.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ portForwarding["Port Forwarding"]
...
+Its main responsibility is to respond to the handshake request from the `host-switch.exe`, create a network namespace and start the `vm-switch` subprocess in the newly created network namespace. In addition, it also calls unshare with provided arguments through `--unshare-args`. Below is a sequence diagram demonstrating the process. The process also establishes a Virtual Ethernet pair consisting of two endpoints: `veth-rd0` and `veth-rd1`. `veth-rd1` resides within the WSL's default namespace and is configured to listen on the IP address `192.168.143.2`. Conversely, `veth-rd0` is located within a network namespace and is assigned the IP address `192.168.143.1`.
```
Comment (mook-as) [inline src/go/networking/README.md:50]: `veth-*` names need to be changed here.

**Review by mook-as — CHANGES_REQUESTED** [review]
Now we need to update even more docs ;)
- https://github.com/rancher-sandbox/rancher-desktop/blob/69ec712707ffd9ad9ef0f676ff1b0877b417f7bd/docs/networking/windows/rancher-desktop-networking.md#L64
(`veth-*` and `192.168.1.*`)
- https://github.com/rancher-sandbox/rancher-desktop/blob/69ec712707ffd9ad9ef0f676ff1b0877b417f7bd/docs/networking/windows/rancher-desktop-networking.md#L86
(`veth-*`)

**Review by mook-as — DISMISSED** [review]

**Review by mook-as — APPROVED** [review]

**Pull request merged**